github_labels.admin.inc

```diff
@@ -11,10 +11,9 @@
 function github_labels_settings_form($form, &$form_state) {
   $form['#config'] = 'github_labels.settings';
 
-  $form[] = array(
+  $form['help'] = array(
+    '#type' => 'help';
```

Comment:
I don't think this should end in a semicolon.

Reply:
Indeed 😅 ...my bad 🤦🏼